Bundle/CoreBundle/Validator/UniquePermalinkValidator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Victoire\Bundle\CoreBundle\Validator;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Victoire\Bundle\ViewReferenceBundle\Connector\ViewReferenceRepository;
+
+class UniquePermalinkValidator extends ConstraintValidator
+{
+    private $viewReferenceRepository;
+    private $requestStack;
+
+    /**
+     * PermalinkValidator constructor.
+     *
+     * @param ViewReferenceRepository $viewReferenceRepository
+     * @param RequestStack            $requestStack
+     */
+    public function __construct(ViewReferenceRepository $viewReferenceRepository, RequestStack $requestStack)
+    {
+        $this->viewReferenceRepository = $viewReferenceRepository;
+        $this->requestStack = $requestStack;
+    }
+
+    /**
+     * @param View       $view
+     * @param Constraint $constraint
+     */
+    public function validate($view, Constraint $constraint)
```

Comment:
This validator will not be called for VirtualBusinessPages, so if a view has a permalink defined, and a VBP url collide with it, it will not be handled

Reply:
Indeed, this is an issue but also an existing issue related to BusinessTemplate : 
- if you create a static page named /project-acme (permalink or not) and I create a BusinessTemplate for a Projet BusinessEntity with an url /project-{{project.slug}}, then the url will override the static page (or vice versa).